gel-cli-instance/src/local/localbackup.rs

```diff
@@ -98,6 +127,14 @@ impl InstanceBackup for LocalBackup {
                 _ = std::fs::remove_dir_all(&temp_dir);
             });
 
+            // TODO: we should trigger the server to start if it's not running
```

Comment:
Yeah, we should. There's no reason not to given that the CLI has complete control over the instance.

Reply:
I want to do this in a follow-up -- coordination of server start/stop adds a bit of complexity and if I'm going to clean up the currently spread-out code that might take some time.